controller-server/src/test/java/com/yahoo/vespa/hosted/controller/deployment/DeploymentTriggerTest.java

```diff
@@ -364,4 +372,94 @@ public void testHandleMultipleNotificationsFromLastJob() {
         assertTrue("All jobs consumed", deploymentQueue.jobs().isEmpty());
     }
 
+    @Test
+    public void dualChangesAreNotSkippedWhenOnePartIsDeployedAlready() {
+        DeploymentTester tester = new DeploymentTester();
+        Application application = tester.createApplication("app1", "tenant1", 1, 1L);
+        Supplier<Application> app = () -> tester.application(application.id());
```

Comment:
Neat. There's about a million other tests that should do this instead. 😄

Reply:
:)